tap_google_sheets/schema.py

```diff
@@ -306,6 +312,14 @@ def get_schemas(client, spreadsheet_id):
                             valid_replication_keys=None,
                             replication_method='FULL_TABLE'
                         )
+                        # for each column check if the `columnSkipped` value is true and the `prior_column_skipped` is false or None
+                        # in the columns dict. The `prior_column_skipped` would be true  when it is the first column of the two
+                        # consecutive empty headers column if true: update the incusion property to `unsupported`
+                        for column in columns:
+                            if column.get('columnSkipped') and not column.get('prior_column_skipped'):
```

Comment:
@namrata270998 What if two consecutive headers are empty?
Eg: `col1,col2,,,col3,col4`

Reply:
@karanpanchal-crest In that case the sheet would be skipped while sync